src/content/docs/linter/rules-sources.mdx

```diff
@@ -6,43 +6,43 @@
 title: Rules sources
 description: A page that maps lint rules from other sources to Biome
 ---
-    
+
 ## Biome exclusive rules
-- [noAccumulatingSpread](/linter/rules/no-accumulating-spread) 
-- [noConsoleLog](/linter/rules/no-console-log) 
-- [noConstEnum](/linter/rules/no-const-enum) 
-- [noDelete](/linter/rules/no-delete) 
-- [noDuplicateObjectKeys](/linter/rules/no-duplicate-object-keys) 
-- [noDynamicNamespaceImportAccess](/linter/rules/no-dynamic-namespace-import-access) 
-- [noEmptyTypeParameters](/linter/rules/no-empty-type-parameters) 
-- [noEnum](/linter/rules/no-enum) 
-- [noEvolvingTypes](/linter/rules/no-evolving-types) 
-- [noExportedImports](/linter/rules/no-exported-imports) 
-- [noGlobalIsFinite](/linter/rules/no-global-is-finite) 
-- [noGlobalIsNan](/linter/rules/no-global-is-nan) 
-- [noImplicitAnyLet](/linter/rules/no-implicit-any-let) 
-- [noInvalidNewBuiltin](/linter/rules/no-invalid-new-builtin) 
-- [noRedundantUseStrict](/linter/rules/no-redundant-use-strict) 
-- [noRenderReturnValue](/linter/rules/no-render-return-value) 
-- [noShoutyConstants](/linter/rules/no-shouty-constants) 
-- [noSuspiciousSemicolonInJsx](/linter/rules/no-suspicious-semicolon-in-jsx) 
-- [noSvgWithoutTitle](/linter/rules/no-svg-without-title) 
-- [noUndeclaredDependencies](/linter/rules/no-undeclared-dependencies) 
-- [noUnnecessaryContinue](/linter/rules/no-unnecessary-continue) 
-- [noUnusedFunctionParameters](/linter/rules/no-unused-function-parameters) 
-- [noUnusedTemplateLiteral](/linter/rules/no-unused-template-literal) 
-- [noUselessStringRaw](/linter/rules/no-useless-string-raw) 
-- [noValueAtRule](/linter/rules/no-value-at-rule) 
-- [noVoidTypeReturn](/linter/rules/no-void-type-return) 
-- [useImportExtensions](/linter/rules/use-import-extensions) 
-- [useNodeAssertStrict](/linter/rules/use-node-assert-strict) 
-- [useShorthandArrayType](/linter/rules/use-shorthand-array-type) 
-- [useSimpleNumberKeys](/linter/rules/use-simple-number-keys) 
-- [useSimplifiedLogicExpression](/linter/rules/use-simplified-logic-expression) 
-- [useSingleCaseStatement](/linter/rules/use-single-case-statement) 
-- [useSortedClasses](/linter/rules/use-sorted-classes) 
-- [useStrictMode](/linter/rules/use-strict-mode) 
-- [useTopLevelRegex](/linter/rules/use-top-level-regex) 
+- [noAccumulatingSpread](/linter/rules/no-accumulating-spread)
```

Comment:
Note: there are some trailing whitespace removals fixes that happened automatically. I noticed here and other files, trailing whitespace is inconsistent, i.e. doesn’t seem to be re-added from formatting.

I can revert these “noisy” changes if needed!